lib/docusign_rest/client.rb

```diff
@@ -1405,5 +1409,57 @@ def add_recipient_tabs(options={})
       response = http.request(request)
       JSON.parse(response.body)
     end
+
+    # Public method - Creates Signing group
+    # group_name: The display name for the signing group. This can be a maximum of 100 characters.
+    # users: An array of group members for the signing group. (see example below)
+    #        It is composed of two elements:
+    #        name – The name for the group member. This can be a maximum of 100 characters.
+    #        email – The email address for the group member. This can be a maximum of 100 characters.
+    # [
+    #   {name: 'test1', email: 'test1@ygrene.us'}
+    #   {name: 'test2', email: 'test2@ygrene.us'}
+    # ]
+    #
+    #
+    # The response returns a success or failure with any error messages.
+    # For successes DocuSign generates a signingGroupId for each group, which is included in the response.
+    # The response also includes information about when the group was created and modified,
+    # including the account user that created and modified the group.
+    def create_signing_group(options={})
+      content_type = { 'Content-Type' => 'application/json' }
+      content_type.merge(options[:headers]) if options[:headers]
```

Comment:
On Line #1457 we are sending `content_type` this will not send custom header passed in.
Please use `merge!` here.